snapcraft/plugins/kbuild.py

```diff
@@ -112,16 +122,60 @@ def __init__(self, name, options, project):
         if logger.isEnabledFor(logging.DEBUG):
             self.make_cmd.append('V=1')
 
+    def unroll_ubuntu_config(self, config_path):
+        try:
+            env = open('debian/debian.env', 'r').read()
+        except OSError as e:
+            raise RuntimeError('Unable to access {}: {}'.format(e.filename,
+                                                                e.strerror))
+        arch = self.project.deb_arch
+        try:
+            branch = env.split('.')[1].strip()
+        except:
```

Comment:
can you just catch `IndexError` here and let the other ones through so we don't mask errors?